### DISCUSSION TITLE
403 Error Again - Riot Changed something?

### DISCUSSION THREAD

**Discussion by HuzPro** [floxay/python-riot-auth#29] [General]
Idk about you guys but I've tried everything and talked to a couple of people and the general consensus is that riot changed something in this whole procedure and it's not working anymore, hasn't been for a bit now, any ideas for possible fixes?

**Comment by floxay** [comment]
Try https://github.com/floxay/python-riot-auth/tree/1.0.9, should work in some regions but not in others.
